cfme/tests/containers/test_containers_projects_summary.py

```diff
@@ -45,8 +51,12 @@ def test_containers_projects_summary_relationships(provider):
             Relationships fields
         """
     navigate_to(Project, 'All')
-    project_name = [r.name.text for r in list_tbl.rows()]
-    for name in project_name:
+    project_names = [r.name.text for r in list_tbl.rows()]
+    project_names_set = set()
```

Comment:
One more optimization here
```project_names = {r.name.text for r in list_tbl.rows()}```
Note the curly braces will cast it into a set and not a list